### ISSUE TITLE
Telemetry options in screen setup should be alphbetical

### ISSUE THREAD

**Issue by jaweber1** [FrSky-OS/discussion#41]
Since you can not "drop down" to see everything, the options in the telemetry screen setup should be alphabetical in order to make searching easier.  Then have a full list with descriptions in the user manual so people can see what's available easily.

**Comment by AloftHobbies** [comment]
The telemetry screens can be adjusted to how the user would like.

To do this, go into Model Setup/Telemetry Set and click the Screen Position button on the lower right of the screen.

**Comment by jaweber1** [comment]
I'd like to reopen this. I think that I was not clear. I understand that the screens can be arranged, what I was referring to is the screen that allows you to pick which data to display. The order of the variables is not alphabetical making it a slow search process to find Curr instead of RxBatt


![telemetry](https://cloud.githubusercontent.com/assets/16408277/11986156/0720466a-a992-11e5-9378-19755381b2cd.jpg)

**Comment by AloftHobbies** [comment]
OK - Good point!!

**Comment by janekx** [comment]
Yes list in alphabetical order will be useful.

**Comment by mac44mag** [comment]
I agree re: alphabetical listing of options.  Perhaps even a "first letter seek" such as we have on our phone address book.
